eta/core/module.py

```diff
@@ -404,19 +476,20 @@ class ModuleMetadata(Configurable, HasBlockDiagram):
     '''Class the encapsulates the architecture of a module.
```

Comment:
Noticed this typo when reviewing this new code.  `Class that` not `Class the`

Reply:
thanks- fixed.